src/main/java/com/rybka/model/Currency.java

```diff
@@ -42,12 +45,12 @@ public void setId(int id) {
         this.id = id;
     }
 
-    public String getCurrencyAbbreviation() {
-        return currencyAbbreviation;
+    public String getBaseCurrencyAbbreviation() {
```

Comment:
Please use Lombok to avoid getter/setter.

Reply:
fixed